README.md

```diff
@@ -22,15 +22,44 @@ This project was created with [`create-next-app`](https://github.com/vercel/next
 
 - nvm [Linux/macOS](https://github.com/nvm-sh/nvm), [Windows](https://github.com/coreybutler/nvm-windows)
 - [yarn](https://classic.yarnpkg.com/en/docs/install)
+- For Windows machines ensure the [VC_redist.x64](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) which is helpful with ensuring issues aren't encountered when trying to run project locally
 
 ### ⚛ Install and Develop
 
 - Clone the repository
 - Run `yarn`
-- Run `nvm use` (or manually add a node version argument when using Windows e.g. `nvm use 14.15.0`)
+- Run `nvm ls` (for Windows to list all currently node versions currently installed)
+- Run `nvm install 16.0.0` (for Windows if node version 16.0.0 hasn't already been previously installed)
```

Comment:
We'll probably want the latest version of 16 which you can get on Windows with `nvm install 16 latest`